packages/statemanager/src/stateManager.ts

```diff
@@ -406,19 +406,28 @@ export class DefaultStateManager implements EVMStateManagerInterface {
    * cache or does a lookup.
    * @private
    */
-  protected _getStorageTrie(address: Address, account: Account): Trie {
-    // from storage cache
-    const addressHex = bytesToUnprefixedHex(address.bytes)
-    const storageTrie = this._storageTries[addressHex]
+  // TODO PR: have a better interface for hashed address pull?
+  protected _getStorageTrie(addressOrHash: Address | Uint8Array, account?: Account): Trie {
```

Comment:
@holgerd77 for your attention and feedback :pray:

Reply:
Hmm, since this is an internal method I would assume this should be fine for now.

Just FYI: we experimented though lately to get rid of this method (and generally: mechanism) completely here #3117 to see if we can get rid of creating all these trie copies.

That's likely (?) another story though.